### ISSUE TITLE
Search: Incorrect tabbing order

### ISSUE THREAD

**Issue by qanitah** [mmiron-HC/WST-RSAMS#54]
URL: https://hcdev.openplus.ca/en/user/203/search
User Agent: Chrome, Firefox
Language: Both

Issue Description: Tabbing order does not match visual order. The button Apply comes after the field Category visually but comes after Home requested with keyboard. 
![issue54](https://user-images.githubusercontent.com/65728700/128895540-edcf2a7c-69c3-442c-ac5e-665b1ea52668.jpg)

Steps to reproduce: Keyboard tabbing on the page.

Expected results: Visual order should match keyboard tabbing order. The button Apply should come at the end after Home requested field visually. It can be moved down on the page to fix the problem. If Apply button is only for the first 3 fields then it can stay where it is and tabbing order should be corrected for it to be after Category field.

**Comment by echouin** [comment]
@qanitah this is now resolved. Tab order is working correctly.

**Comment by hydegeoff** [comment]
Issue fixed, closed.
